### ISSUE TITLE
Same token in request after refresh token

### ISSUE THREAD

**Issue by israelKusayev** [Flyrell/axios-auth-refresh#54]
The interceptor is working fine. but in some post requests the second request is sent with the previous access token

here is the refresh token logic.
```javascript
// Function that will be called to refresh authorization
const refreshAuthLogic = (failedRequest: any) =>
    authService
        .refreshToken(authService.getRefreshToken()!)
        .then(res => {
            authService.storeTokens(res.data);
            return Promise.resolve();
        })
        .catch(() => {
            authService.logout();
            history.push('/login');
            return Promise.reject();
        });

// Instantiate the interceptor (you can chain it as it returns the axios instance)
createAuthRefreshInterceptor(axios, refreshAuthLogic);
```

and I have axios interceptor that should set the header each request
```javascript
// Add a request interceptor
axios.interceptors.request.use(function (config) {
    config.headers.Authorization = 'Bearer ' + authService.getAccessToken();
    return config;
});
```

![image](https://user-images.githubusercontent.com/42112796/70918865-79a72800-2028-11ea-8ac7-0db0600e6f44.png)

**Comment by Flyrell** [comment]
Hi, can you please check if your request interceptor that handles Authorization header is triggered when  those failing requests are called?

**Comment by israelKusayev** [comment]
You right the interceptor isn't trigger  
Is it OK, or it's a bug in your interceptor?

**Comment by Flyrell** [comment]
This is probably a bug, which has also been discussed in #40. I will fix it ASAP.

**Comment by Flyrell** [comment]
This should no be resolved in v2.0.2. Can you confirm please?

**Comment by israelKusayev** [comment]
Works OK! thank you
